.github/workflows/eamxx-gh-ci-standalone.yml

```diff
@@ -0,0 +1,55 @@
+name: gh-standalone
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      # first, yes to these
+      - '.github/workflows/eamxx-gh-ci-standalone.yml'
+      - 'cime_config/machine/config_machines.xml'
+      - 'components/eamxx/**'
+      - 'components/homme/**'
+      # second, no to these
+      - '!components/eamxx/docs/**'
+      - '!components/eamxx/mkdocs.yml'
+
+  workflow_dispatch:
+
+jobs:
+
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - sp
+          - opt
+          - dbg
+          - fpe
+    container: 
+      image: ghcr.io/e3sm-project/containers-standalone-ghci:standalone-ghci-0.1.0
+
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          submodules: recursive
+      - 
+        name: standalone
+        env:
+          SHELL: sh
+        run: |
+          # TODO: get rid of this extra line if we can?
+          git config --global safe.directory '*'
+          ./components/eamxx/scripts/test-all-scream -m ghci-oci -t ${{ matrix.test }}
+      - 
+        name: Artifacts
+        uses: actions/upload-artifact@v4
+        if: ${{ always() }}
+        with:
+          name: ${{ matrix.test }}
```

Comment:
I am ok with short names for actions names, since they appear at the bottom of the PR page, and long names are problematic. But artifacts are in the action page, and there's plenty of space for a longer name. That said, I think it's very subjective, hence the approval regardless.